scripts/start_herb.sh

```diff
@@ -16,7 +16,8 @@ function pr_launcher {
     launcher $1 "roslaunch herb_launch $2 --wait"
 }
 
-if [ $(hostname) != 'herb0.personalrobotics.ri.cmu.edu' ]
+# TODO make herb0's hostname actually .personalrobotics.ri.cmu.edu
+if [ $(hostname) != 'herb0' ]
```

Comment:
Actually, I think checking if the hostname starts with `herb0.*` is probably better because the FQDN could change (and probably will at UW).

Maybe something like:
`if [[ $(hostname) != 'herb0.'* ]]`?

http://stackoverflow.com/a/4132551